frontend/pages/DashboardPage/cards/MDM/MDM.tsx

```diff
@@ -65,6 +65,28 @@ const EmptyMdmSolutions = (): JSX.Element => (
   />
 );
 
+type IMdmSolutionMap = Record<string, IMdmSolution>;
+
+const reduceSolutionsToObj = (mdmSolutions: IMdmSolution[]) => {
+  return mdmSolutions.reduce<IMdmSolutionMap>((acc, nextSolution) => {
+    // The solution name can be null so we add an Unknown key to the
+    // accumulator in this case.
+    if (nextSolution.name === null) {
```

Comment:
Suggestion:

```tsx

const key = nextSolution.name ? "Unknown" : nextSolution.name;
if (acc[key]) {
      acc[key].hosts_count += nextSolution.hosts_count;
} else {
      acc[key] = Object.assign({ ...nextSolution });
}
```

Reply:
looks good just need to switch around the ternary to

`const key = nextSolution.name ? nextSolution.name : "Unknown";`